### PR TITLE
Add query filtering

### PR DESCRIPTION
**Pull request by rlecomte**
## What this PR does

Support query params filtering.

## Related

See: #61

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -18,9 +18,11 @@ pub fn create_loader_runtime() -> Result<tokio::runtime::Runtime, Error> {
         .context("Error creating loader tokio runtime")
 }
 
-pub fn start_logger() {
+pub fn start_logger() -> Result<(), Error> {
     let env = Env::new().filter_or("RUST_LOG", "INFO");
-    env_logger::init_from_env(env);
+    env_logger::try_init_from_env(env)
```
Comment (fteychene) [inline src/lib.rs:23]: Just curious : In which case had you the error ?

File: Cargo.toml [hunk 2]
```diff
@@ -24,4 +24,5 @@ structopt = "0.3"
 signal-hook = "0.1"
 reqwest = { version = "0.10", features = ["blocking"] }
 get-port = "1.3.1"
-lazy_static = "1.4.0"
\ No newline at end of file
+lazy_static = "1.4.0"
```
Comment (fteychene) [inline Cargo.toml:27]: Where is this crate used ?

File: src/lib.rs (hunk 1 above)
Comment (rlecomte) [reply src/lib.rs:23]: In the integration test. I added log to have a best feedback on what's gone wrong.

Comment (rlecomte) [reply src/lib.rs:23]: as we launch that multiple time, it fail after the first time we call it.

File: Cargo.toml (hunk 2 above)
Comment (rlecomte) [reply Cargo.toml:27]: this crate was already there

File: src/lib.rs (hunk 1 above)
Comment (fteychene) [reply src/lib.rs:23]: Ok for this PR we should change this behavior in another PR :+1:

File: Cargo.toml (hunk 2 above)
Comment (fteychene) [reply Cargo.toml:27]: Ok it's a miss going to remove it on a cleanup PR :+1:

**Pull request merged**